code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1719,14 +1719,13 @@
 
 
 /mob/living/carbon/human/take_over(mob/M)
-	assigned_squad?.clean_marine_from_squad(src)
+	if(assigned_squad)
+		assigned_squad.clean_marine_from_squad(src)
```

Comment:
Why the format change? The prior one worked the same, but with one less check.

Reply:
Because I don't trust it :cry:

Reply:
Stop being paranoid.
D=<
This should be changing nothing.

Reply:
i think the new code is more readible

Reply:
Reeeeee.
Two against one, sad times. Oh well.
If you prefer it for readability I won't pry further, just don't assume it doesn't do what it's supposed to do.